### PR TITLE
chore: add Kubernetes patch releases to VHD

### PR DESCRIPTION
**Pull request by deaborch**
**Reason for Change**:

Added support for 1.14.8 and 1.15.5 on Azure Stack

**Issue Fixed**:

**Requirements**:

- [X] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [X] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

File: packer/configure-windows-vhd.ps1 [hunk 1]
```diff
@@ -67,6 +67,7 @@ function Get-FilesToCacheOnVHD
         "c:\akse-cache\win-k8s\" = @(
             "https://acs-mirror.azureedge.net/wink8s/azs-v1.14.6-1int.zip",
             "https://acs-mirror.azureedge.net/wink8s/azs-v1.14.7-1int.zip",
+            "https://acs-mirror.azureedge.net/wink8s/azs-v1.14.8-1int.zip",
```
Comment (jackfrancis) [inline packer/configure-windows-vhd.ps1]: Looks like we already had the 1.15.5 artifact config here?

Comment (deaborch) [reply packer/configure-windows-vhd.ps1]: Actually forgot to add it, thanks

File: docs/topics/azure-stack.md [hunk 2]
```diff
@@ -190,7 +190,7 @@ _Note: AKS Engine on disconnected Azure Stack instances is a preview feature_
...
 | v0.39.2   | [AKS Base Ubuntu 16.04-LTS Image Distro, August 2019 (2019.08.09)](https://github.com/Azure/aks-engine/blob/v0.39.1/releases/vhd-notes/aks-ubuntu-1604/aks-ubuntu-1604-201908_2019.08.09.txt) | 1.15.2 - 1.14.5 - 1.13.9 - 1.12.8 - 1.12.7 - 1.11.10 - 1.11.9 |  |
 | v0.40.1   | [AKS Base Ubuntu 16.04-LTS Image Distro, August 2019 (2019.08.21)](https://github.com/Azure/aks-engine/blob/v0.40.0/releases/vhd-notes/aks-ubuntu-1604/aks-ubuntu-1604-201908_2019.08.21.txt) | 1.15.3 - 1.15.2 - 1.14.6 - 1.14.5 - 1.13.10 - 1.13.9 - 1.12.8 - 1.12.7 - 1.11.10 - 1.11.9 |  |
-| v0.41.2   | [AKS Base Ubuntu 16.04-LTS Image Distro, September 2019 (2019.09.19)](https://github.com/Azure/aks-engine/blob/v0.41.2/releases/vhd-notes/aks-ubuntu-1604/aks-ubuntu-1604-201908_2019.09.19.txt) | 1.15.4 - 1.15.3 - 1.14.7 - 1.14.6 - 1.13.11 -  1.13.10 |  |
+| v0.41.2   | [AKS Base Ubuntu 16.04-LTS Image Distro, September 2019 (2019.09.19)](https://github.com/Azure/aks-engine/blob/v0.41.2/releases/vhd-notes/aks-ubuntu-1604/aks-ubuntu-1604-201908_2019.09.19.txt) | 1.15.5 - 1.15.4 - 1.14.8 - 1.14.7 - 1.13.11 -  1.13.10 |  |
```
Comment (rjaini) [inline docs/topics/azure-stack.md]: We can't change this table as it is for old release. 
Once new VHD is out then we will add new row.

File: packer/install-dependencies.sh [hunk 3]
```diff
@@ -400,9 +400,11 @@ K8S_VERSIONS="
...
 1.16.1-azs
 1.15.5
+1.15.5-azs
 1.15.4
 1.15.4-azs
 1.14.8
+1.14.8-azs
 1.14.7
 1.14.7-azs
 1.13.12
```
Comment (jadarsie) [inline packer/install-dependencies.sh]: Let's remove `1.13.11-azs`

**Comment by jadarsie** [comment]
@jackfrancis @mboersma cannot retrigger the 1.13 e2e, can we call this good regardless? these are VHD related changes only

**Pull request merged**